docs/antora.yml

```diff
@@ -0,0 +1,5 @@
+name: lsp
+title: pkl-lsp
+version: 0.1.1
```

Comment:
Current version is 0.1.0

Reply:
Yeah, but the version on main is 0.1.1 (see https://github.com/apple/pkl-lsp/blob/689fb86fa24bdf2aa62d170b491704987045ca0c/gradle.properties#L2)